lib/sectorbuilder/sectorbuilder_test.go

```diff
@@ -90,25 +72,73 @@ func TestSealAndVerify(t *testing.T) {
 		t.Fatal("proof failed to validate")
 	}
 
+	done()
+}
+
+func (s *seal) post(t *testing.T, sb *sectorbuilder.SectorBuilder) {
 	cSeed := [32]byte{0, 9, 2, 7, 6, 5, 4, 3, 2, 1, 0, 9, 8, 7, 6, 45, 3, 2, 1, 0, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0, 9}
 
 	ssi := sectorbuilder.NewSortedSectorInfo([]sectorbuilder.SectorInfo{{
-		SectorID: sid,
-		CommR:    pco.CommR,
+		SectorID: s.sid,
+		CommR:    s.pco.CommR,
 	}})
 
 	postProof, err := sb.GeneratePoSt(ssi, cSeed, []uint64{})
 	if err != nil {
 		t.Fatalf("%+v", err)
 	}
 
-	ok, err = sectorbuilder.VerifyPost(context.TODO(), sb.SectorSize(), ssi, cSeed, postProof, []uint64{})
+	ok, err := sectorbuilder.VerifyPost(context.TODO(), sb.SectorSize(), ssi, cSeed, postProof, []uint64{})
 	if err != nil {
 		t.Fatalf("%+v", err)
 	}
 	if !ok {
 		t.Fatal("bad post")
 	}
+}
+
+func TestSealAndVerify(t *testing.T) {
+	//t.Skip("this is slow")
```

Comment:
Reskip the tests, or skip if `GOMAXPROCS < 10` (so it doesn't burn my notebook).